datareservoirio/client.py

```diff
@@ -250,12 +250,20 @@ def wrapper(self, series_id, start=None, end=None, **kwargs):
             result = func(self, series_id, start=start, end=end, **kwargs)
             end_time = time.perf_counter()
             elapsed_time = end_time - start_time
+            start_date_as_str = pd.to_datetime(
+                start, dayfirst=True, unit="ns", utc=True
+            ).isoformat()
+            end_date_as_str = pd.to_datetime(
+                end, dayfirst=True, unit="ns", utc=True
+            ).isoformat()
```

Comment:
Will not work when `end=None`

Reply:
Possible solution: 
![image](https://user-images.githubusercontent.com/68221790/228264508-df59e7a4-9891-410a-8cb6-ca289cb30d45.png)